_data/toc.yaml

```diff
@@ -1396,6 +1394,40 @@ manuals:
           title: Working with Buildx
         - path: /build/buildx/install/
           title: Install Buildx
+    - sectiontitle: Bake
+      section:
+        - path: /build/bake/
+          title: Overview
+        - path: /build/bake/file-definition/
+          title: File definition
+        - path: /build/bake/configuring-build/
+          title: Configuring builds
+        - path: /build/bake/hcl-funcs/
+          title: User defined HCL functions
+        - path: /build/bake/build-contexts/
+          title: Build contexts and linking targets
+        - path: /build/bake/compose-file/
+          title: Building from Compose file
+    - sectiontitle: Guides
```

Comment:
Ahh this seems strange to me, but not sure whether there's a better alternative.

Having a section in Manuals called Guides, when we have a Guides category :thinking: I feel like it's better to maybe be consistent with the rest of our docs for now, and put it into Guides?

Other options:
- Don't call them "Guides" but something like "Advanced usage"? But then of course, what happens if we add more entry-level guides/tutorials, do those go into the guides section (splitting them up), or do we keep them together, which seems inconsistent with the rest of the docs.
- Rework the buildkit guides to be more like the guides in the rest of the docs, by making them more tutorial like, etc - maybe the [remote builder guide](https://github.com/docker/buildx/blob/master/docs/guides/remote-builder.md) is already like this in style?